routes/artifacts.js

```diff
@@ -18,7 +18,7 @@ api.declare({
       'assume:worker-id:<workerGroup>/<workerId>'
     ], [
       'queue:create-artifact:<name>',
-      'queue:claim-task:<taskId>/<runId>'
+      'queue:claim-run-id:<taskId>/<runId>'
```

Comment:
Better than reusing `queue:claim-run-id:<taskId>/<runId>` would be `queue:create-artifact-for:<taskId>/<runId>` - otherwise we assume that whichever component is allowed to claim a task is also allowed to create an artifact for it. Imagine a fictitious "Jenkins Bridge" component whereby a central component claimed tasks, but individual Jenkins slaves posted artifacts. The roles would indeed be separate.